### ISSUE TITLE
GitHub Actions - Publish Github release

### ISSUE THREAD

**Issue by xRadne** [strusoft/femdesign-api#352]
We could create a GitHub action to automatically publish a release to Github with FemDesign.Grasshopper and FemDesign.Dynamo assets attached. This could be done similar to the way the NuGet packages are auto-deployed. 

Example: When the `GlobalAssemblyInfo.cs` file (or better when the Assembly version) is changed on the master branch (or pre-release/**) a new release should be deployed.

Maybe for 21.1.1? #338

**Comment by andosca** [comment]
Maybe for 21.2.0 :)

**Comment by xRadne** [comment]
@andosca @Marco-Pellegrino Maybe all examples should be zipped and included with each auto-release?

However, this might cause some problems if an example gets out-of-date but still gets included. This might cause more confusion than it has benefits. If we instead only try to make the examples up-to-date with the latest release that should be sufficient for most users.

**Comment by Marco-Pellegrino** [comment]
@xRadne 
I agree. We should give some example to the users when they download the tool. The .zip file can have the latest release + the examples folder.

**Comment by xRadne** [comment]
Not possible before #387
